helm-charts/basehub/values.jsonnet

```diff
@@ -68,6 +68,9 @@ local jupyterhubHomeNFSConfig = {
       QuotaManager: {
         paths: ['/export/%s' % hub_name],
         hard_quota: 0,
+        projects_file: '/export/%s/.projects' % hub_name,
+        projid_file: '/export/%s/.projid' % hub_name,
```

Comment:
Now we're going to store these quota files on the NFS volume. 

These files do not _need_ to be persisted, but by placing them here we can make restarts faster by avoiding state invalidation.